docs/docs/core/06-grpc_rest.md

```diff
@@ -78,15 +78,16 @@ The Cosmos SDK's [Swagger generation script](https://github.com/cosmos/cosmos-sd
 
 ## CometBFT RPC
 
-Independently from the Cosmos SDK, CometBFT also exposes a RPC server. This RPC server can be configured by tuning parameters under the `rpc` table in the `~/.simapp/config/config.toml`, the default listening address is `tcp://localhost:26657`. An OpenAPI specification of all CometBFT RPC endpoints is available [here](https://docs.cometbft.com/master/rpc/).
+Independently from the Cosmos SDK, CometBFT also exposes a RPC server. This RPC server can be configured by tuning parameters under the `rpc` table in the `~/.simapp/config/config.toml`, the default listening address is `tcp://localhost:26657`. An OpenAPI specification of all CometBFT RPC endpoints is available [here](https://docs.cometbft.com/main/rpc/).
 
 Some CometBFT RPC endpoints are directly related to the Cosmos SDK:
 
 * `/abci_query`: this endpoint will query the application for state. As the `path` parameter, you can send the following strings:
     * any Protobuf fully-qualified service method, such as `/cosmos.bank.v1beta1.Query/AllBalances`. The `data` field should then include the method's request parameter(s) encoded as bytes using Protobuf.
     * `/app/simulate`: this will simulate a transaction, and return some information such as gas used.
     * `/app/version`: this will return the application's version.
-    * `/store/{path}`: this will query the store directly.
+    * `/store/{storeName}/key`: this will directly query the named store for data associated with the key represented in the `data` parameter.
+    * `/store/{storeName}/subspace`: this will directly query the named store for key/value pairs in which the key has the value of the `data` parameter as a prefix.
```

Comment:
Is this really a CometBFT RPC endpoint?
The only place I have seen a mention of it is here: https://github.com/cosmos/cosmos-sdk/blob/b9b325ec991a095cecc719070b7c724ae7cc9134/store/iavl/store.go#L354